micro-ui/web/micro-ui-internals/packages/modules/pgr/src/components/PGRCard.js

```
@@ -0,0 +1,75 @@
+  const { t } = useTranslation();
+
+  const allLinks = [
+    { text: t("ES_PGR_INBOX"), link: `/${window?.contextPath}/employee/pgr/inbox` },
+    { text: t("ES_PGR_NEW_COMPLAINT"), link: `/${window?.contextPath}/employee/pgr/complaint/create`, accessTo: ["CSR"] },
+  ];
+
+  if (!Digit.Utils.pgrAccess()) {
+    return null;
+  }
+
+  const Icon = () => <svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24">
```

Biome

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)